subsys/mgmt/hawkbit/hawkbit.c

```diff
@@ -805,8 +804,8 @@ static void response_cb(struct http_response *rsp,
 		}
 
 		if (body_data != NULL) {
-			ret = flash_img_buffered_write(&hb_context.flash_ctx,
```

Comment:
Can we move whitespace changes to a separate commit, if they are necessary?

Reply:
It appears to be in a number of places. Would help to make it clearer where the relevant change is located.

Reply:
@cfriedt This PR has 2 commits. One of them `mgmt: hawkbit: fix codestyle and whitespaces` is only to fix codestyle. Do you want me to open a new PR only to fix codestyle?